flink-streaming-scala/src/main/scala/org/apache/flink/streaming/api/scala/StreamExecutionEnvironment.scala

```diff
@@ -124,7 +125,7 @@ class StreamExecutionEnvironment(javaEnv: JavaEnv) {
    * Gets the checkpoint config, which defines values like checkpoint interval, delay between
    * checkpoints, etc.
    */
-  def getCheckpointConfig = javaEnv.getCheckpointConfig()
```

Comment:
Calling Java methods with parenthesis or without actually somewhat subject to debate.

For Scala methods, the writer decides whether it is a method with or without parenthesis.
For Java, the caller can only "guess" what the method does (is returning a mutable field considered a side effect?), so in most places we call Java methods as Java methods.